plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/CoinbaseDownloader.java

```diff
@@ -282,50 +281,30 @@ private List<UserTrade> downloadTrades(Map<String, WalletState> walletStates) th
             final String walletId = entry.getKey();
             final WalletState walletState = wallets.get(walletId);
             if (walletRequests < MAX_WALLET_REQUESTS) {
-                String lastBuyId = entry.getValue().lastBuyId;
-                String lastSellId = entry.getValue().lastSellId;
```

Comment:
Some clients downloaded the connector and the WalletState for buy/sell was stored separately as String. Now you remove the lastSellId. How do you prevent users from having any duplicate downloads of sell transactions?